### PR TITLE
Make the reorganized contents available for zh-cn index.

### PR DESCRIPTION
**Pull request by dongxuwang**

### CODE REVIEW

**Comment by SethTisue** [comment]
I don't know how this aspect of the site works. Are you confident in this, should I just hit "merge"?

**Comment by b-studios** [comment]
I am also not confident about this, yet. Using another template might break things in the english version.

**Comment by dongxuwang** [comment]
Don't worry, guys. I have tested locally.

**Comment by b-studios** [comment]
If I understand correctly, you are copy and pasting the english contents to the other languages index pages, right? Couldn't that be a bit misleading if people look for docs in say Russian? Also this implies that for every change we make on the index page in english, we also need to adjust the copies in the other languages.

@SethTisue I know you said you are no expert on the multilingual part, but neither am I. :) Do you think the duplication is worth it?

**Comment by b-studios** [comment]
An alternative would be to have some sort of disclaimer that says: "These contents are currently only available in english. Once we are stable, we are happy for contributions in other languages."

**Comment by SethTisue** [comment]
@b-studios sorry, no opinion

**Comment by dongxuwang** [comment]
Hmm, actually, this pr is to adjust the layout of the three indices. The scala 3 part has not been translated, thus copy and paste.

**Comment by dongxuwang** [comment]
I changed the pr to fix only the zh-cn's layout with translated content.

**Comment by SethTisue** [comment]
is someone able to review and/or merge this?

**Comment by b-studios** [comment]
I don't speak the language, but I can of course merge ;)

**Comment by dongxuwang** [comment]
Can I do it myself?

File: _zh-cn/index.md [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+    - title: "Scala 3 书籍"
+      description: "一部介绍主要语言特性的线上书"
+      icon: "fa fa-book"
+      link: /scala3/book/introduction.html
+- title: "更多细节"
+  links:
+    - title: "迁移指引"
+      description: "一份帮你从 Scala 2 迁移到 Scala 3 的指引"
+      icon: "fa fa-suitcase"
+      link: https://scalacenter.github.io/scala-3-migration-guide/
```
Comment (b-studios) [inline _zh-cn/index.md:31]:
```suggestion
      link: /scala3/guides/migration/compatibility-intro.html
```

Comment (b-studios) [reply _zh-cn/index.md:31]: This link changed in the meantime.

**Comment by b-studios** [comment]
I'll merge once you accept the changed URL which changed in the meantime.

**Pull request merged**